frontend/src/charts/TableChart.tsx

```diff
@@ -190,7 +175,7 @@ export function TableChart(props: TableChartProps) {
         <figure>
           <figcaption>
             <ChartTitle
-              title={`Breakdown summary for ${sentenceCaseName} in ${props.fips.getSentenceDisplayName()}`}
+              title={`${props.dataTableTitle} in ${props.fips.getSentenceDisplayName()}`}
```

Comment:
it's possible this could be sent in as an empty string right, so the title would read ` in Texas` or something; maybe instead of empty string as a fallback in the parent we could use a generic term like `Data`. I'll add a note up there